examples/bot.py

```diff
@@ -66,5 +71,6 @@ async def hello_world(ctx: interactions.SlashContext):
 #   object on line 13.
 # - we are not setting a presence.
 # - we are not automatically sharding, and registering the connection under 1 shard.
-# - we are using default intents, which are Gateway intents excluding privileged ones.
+# - we are using default intents and the message content intent, which are Gateway intents and
+# - message content excluding privileged ones.
```

Comment:
This is... confusing. It makes it sound like message content *isn't* a privileged intent - this sentence may need a total rewording to make more sense.

Reply:
I reworded the comments to try and make it make more sense, and explicitly say that message content is a privileged intent